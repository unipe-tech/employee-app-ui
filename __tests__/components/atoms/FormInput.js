import { render, fireEvent, waitFor } from "@testing-library/react-native";
import { describe, expect, test } from "@jest/globals";
import FormInput from "../../../components/atoms/FormInput";

describe("FormInput", () => {
  test("renders UI correctly", () => {
    const tree = render(<FormInput />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
