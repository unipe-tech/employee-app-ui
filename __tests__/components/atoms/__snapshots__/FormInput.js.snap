// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormInput renders UI correctly 1`] = `
<View
  style={Object {}}
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "borderColor": "#aaaaaa",
        "borderRadius": 5,
        "borderWidth": 1.5,
        "flexDirection": "row",
        "height": 50,
        "paddingHorizontal": 19.736842105263158,
        "width": "100%",
      }
    }
  >
    <TextInput
      autoCompleteType="off"
      editable={true}
      keyboardType="default"
      onChangeText={[Function]}
      placeholderTextColor="#808080"
      style={
        Object {
          "color": "#000000",
          "flex": 1,
          "fontFamily": "Montserrat-Regular",
          "fontSize": 14,
          "lineHeight": 18,
        }
      }
    />
  </View>
</View>
`;
